layers/+web/eww/packages.el

```diff
@@ -32,9 +32,9 @@
     :defer t
     :init
     (spacemacs//eww-setup-transient-state)
-    (spacemacs/declare-prefix "ate" "eww")
-    (spacemacs/set-leader-keys "atee" 'eww)
-    (spacemacs/set-leader-keys "atew" 'eww-switch-to-buffer)
+    (spacemacs/declare-prefix "awe" "eww")
```

Comment:
you also need to update the commit message and documentation.

Reply:
I've updated the documentation, but I already put this in a separate commit, incl. commit message, for clarity of history (this PR has two commits)

Reply:
Regarding the commits, I prefer to have the change of eww keybindings in the primary commit message, therefore I prefer two commits.